en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -274,15 +279,15 @@ To deploy the API, follow the steps below:
 
     ![Deployment progress](../assets/img/tutorials/rest-api/deployment-progress.png){.cInlineImage-full}
 
-Once the API is deployed, the **Development** card indicates that the API is running as shown below.
+Once the API is deployed, the **Development** card indicates that the API is active as shown below.
 
 ![Deployed API](../assets/img/tutorials/rest-api/deployed-api.png){.cInlineImage-full}
 
 Now you can test your deployed REST API to check again whether it is working as expected.
 
 ## Step 3: Test
 
-Once you have deployed the Choreo application you can test it via the Open API Console, A cURL command, or Postman.
+Once you have deployed the Choreo application you can test it via the Open API Console, a cURL command, or Postman.
```

Comment:
The correct term is "OpenAPI Console". Might have to fix this in all applicable places in the doc.

Reply:
Checked and found that it is used in two places, and corrected it in both the places.